requirements.txt

```diff
@@ -35,3 +35,4 @@ waitress==0.8.10          # via webtest
 webob==1.4.1              # via webtest
 webtest==2.0.18           # via django-webtest
 wmctrl==0.2               # via pdbpp
+selenium
```

Comment:
Stick this in `requirements.in` and regenerate with `pip-compile`. See https://github.com/nvie/pip-tools#example-usage-for-pip-compile